plugins/covid/lab.py

```diff
@@ -145,3 +213,48 @@ def positive(test):
     covid_test = _get_covid_test(test)
     result = get_result(test)
     return result in covid_test.POSITIVE_RESULTS
+
+
+def get_covid_result_ticker(patient):
+    """
+    Given a PATIENT, return a list of dictionaries representing the last
+    3 results for each test type, with a timestamp, result string, and test name.
+    """
+    ticker = []
+
+    for test in COVID_19_TESTS:
+        data = []
+
+        tests = lab_test_models.LabTest.objects.filter(
+            test_name=test.TEST_NAME, patient=patient
+        ).order_by('-datetime_ordered')
+
+        for test in tests:
+            if len(data) > 2:
+                continue
```

Comment:
break rather than continue is the behaviour we want? (Although no real difference)